SI/Sem.03/Pract.03/Prakt3Task11.cpp

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+#include <cmath>
+using namespace std;
+int main()
+{
+    cout << "Enter two numbers: "<< endl;
+    int a, b, temp;
+    bool isPrime;
+    cin >> a >> b;
+    if (a > b)
+    {
+        temp = a;
+        a = b;
+        b = temp;
+    }
+    for(int i = a; i <= b; i++) 
+    {
+        isPrime = true;
+        for (int j = 2; j <= sqrt(i); j++)
```

Comment:
Never use _sqrt()_ in a loop condition because it calculates the square root at every iteration and this makes the program slower.